packages/eth-providers/src/consts.ts

```diff
@@ -87,7 +87,8 @@ export const ORPHAN_TX_DEFAULT_INFO = {
   ...DUMMY_V_R_S,
 };
 
-export const BLOCK_GAS_LIMIT = 29_990_102;
+export const BLOCK_GAS_LIMIT = 29_990_016;
+export const BLOCK_STORAGE_LIMIT = 3_670_016;
```

Comment:
https://github.com/AcalaNetwork/Acala/blob/dfbda3a4cc3d6b3ea78bdfdf542b3c2d7fc5d576/runtime/integration-tests/src/evm.rs#L1158-L1159

Reply:
We should get this parameter from the chain later